opam-format.opam

```diff
@@ -17,17 +17,18 @@ authors: [
   "Ralf Treinen <ralf.treinen@pps.jussieu.fr>"
   "Frederic Tuong <tuong@users.gforge.inria.fr>"
 ]
-homepage: "https://opam.ocaml.org/"
+homepage: "https://opam.ocaml.org"
 bug-reports: "https://github.com/ocaml/opam/issues"
 dev-repo: "git+https://github.com/ocaml/opam.git"
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
 build: [
   ["./configure" "--disable-checks" "--prefix" prefix]
   ["dune" "build" "-p" name "-j" jobs]
 ]
 depends: [
   "ocaml" {>= "4.02.3"}
   "opam-core" {= version}
-  "opam-file-format" {>= "2.1.3"}
-  "re" {>= "1.9.0"}
-  "dune" {>= "1.11.0"}
+  "opam-file-format" {>= "2.0.0~rc2"}
```

Comment:
I tweaked this one since technically opam still builds with the old opam-file-format, it just can't "read" opam 2.1+ files without it. The other dependencies had got pulled through in the cherry-pick